hw/bsp/d5035_01/board.mk

```diff
@@ -0,0 +1,63 @@
+CFLAGS += \
+  -mthumb \
+  -mabi=aapcs \
+  -mlong-calls \
+  -mcpu=cortex-m4 \
+  -mfloat-abi=hard \
+  -mfpu=fpv4-sp-d16 \
+  -nostdlib -nostartfiles \
+  -D__SAME51J19A__ \
+  -DCONF_CPU_FREQUENCY=80000000 \
+  -DCONF_GCLK_USB_FREQUENCY=48000000 \
+  -DCFG_TUSB_MCU=OPT_MCU_SAME5X \
+  -DD5035_01=1 \
+  -DBOARD_NAME="\"D5035-01\"" \
+  -DSVC_Handler=SVCall_Handler
```

Comment:
thanks, I think this is mostly the symbol difference because of the startup asm code. SAMD51 used the modified/unified version from adafruit https://github.com/adafruit/samd-peripherals . By now, I just realized adafruit just adding same5 support to the repo. But that should be no problems, the tinyusb is a neutral library, application is free to pick any other driver/libraries they like :)